freight/api/config.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+
+from flask import current_app, redirect, url_for
+
+from freight import get_version
+from freight.api.base import ApiView
+
+from urlparse import urlparse
+
+
+class ConfigApiView(ApiView):
+    def get(self):
+        """
+        Returns a configuration object
+        """
+
+        if not self.is_authorized():
```

Comment:
How does auth work locally? I tested it locally in incognito and it still views me as logged in O.o